wfe/web-front-end.go

```diff
@@ -981,12 +989,14 @@ func (wfe *WebFrontEndImpl) Certificate(response http.ResponseWriter, request *h
 		if !strings.HasPrefix(path, CertPath) {
 			logEvent.Error = "Certificate not found"
 			wfe.sendError(response, logEvent.Error, path, http.StatusNotFound)
+			response.Header().Add("Cache-Control", "public, max-age=0, no-cache")
```

Comment:
This line is repeated many times. Can you factor our a `addNoCacheHeader(response)` function?